hail/src/main/scala/is/hail/expr/ir/Interpretable.scala

```diff
@@ -1,11 +1,12 @@
 package is.hail.expr.ir
 
-import is.hail.types.virtual.TNDArray
+import is.hail.types.virtual.{TNDArray, TStream}
 
 object Interpretable {
   def apply(ir: IR): Boolean = {
     !ir.typ.isInstanceOf[TNDArray] &&
-      (ir match {
+      !ir.typ.isInstanceOf[TStream] &&
```

Comment:
Why this? There are interpret rules for most of the stream nodes.

Reply:
not sure, actually. will revert.